Library/Homebrew/cmd/install.rb

```diff
@@ -47,6 +47,10 @@ class InstallCmd < AbstractCommand
                description: "Ask for confirmation before downloading and installing formulae. " \
                             "Print download and install sizes of bottles and dependencies.",
                env:         :ask
+        switch "--non-interactive-sudo",
+               description: "Fail fast on any interactive sudo prompt and use non-interactive sudo where possible."
+        flag "--timeout-wait-for-user=",
+             description: "Wait this many seconds when an interactive prompt is detected; then skip the item."
```

Comment:
@rehkad Which of these options are you personally needing/using? Both?

I think it may still make sense to use the `env:` argument here as, correct me if I'm wrong, this is probably something you want to set somewhat globally?